needls/server.py

```diff
@@ -334,9 +334,12 @@ def completions(ls, params: CompletionParams = None):
     if not ls.needs_store.needs_initialized:
         return []
 
-    lines, word = get_lines_and_word(ls, params)
-    line_number = params.position[0]
-    line = lines[line_number]
+    try:
+        lines, word = get_lines_and_word(ls, params)
+        line_number = params.position[0]
+        line = lines[line_number]
+    except IndexError:
```

Comment:
Missing a comment here, why ignoring an IndexError is a good idea or even why this is happening.